code/game/objects/structures/arclumin.dm

```diff
@@ -0,0 +1,21 @@
+/*
+
+At 7:30 AM on March 8th, 2017, ArcLumin died in a fatal car crash on impact.
+Rest in peace, man. You did good work.
+When a coder for SS13 dies, all codebases feel it and suffer.
+We may disagree on whether farts should be a thing, or what color to paint the bikeshed,
+but we are all coders together.
+
+Goodbye, man. We'll miss you.
+
+This memorial has been designed for him and any future coders to perish.
+
+*/
+
+/obj/structure/arc
+	name = "Tomb of the Unknown Spaceman"
+	desc = "Here rests an unknown spaceman\nUnknown by name or rank\nWhose acts will not be forgotten"
```

Comment:
close, it's

`Here Rests\nAn unknown employee\nUnknown by name or rank\nWhose acts will not be forgotten`